Apps/W1/Email - SMTP Connector/app/src/SMTPMessage.Codeunit.al

```diff
@@ -101,10 +101,22 @@ codeunit 4514 "SMTP Message"
         EmailRecipientType: Enum "Email Recipient Type";
         AttachmentInStream: InStream;
         Body: Text;
+        FromName: Text[250];
+        FromAddress: Text[250];
         Recipients: List of [Text];
     begin
         // Add From
-        AddFrom(SMTPAccount."Sender Name", SMTPAccount."Email Address");
+        //AddFrom(SMTPAccount."Sender Name", SMTPAccount."Email Address");
+        if Message.GetFromAddress() <> '' then
```

Comment:
Looks good, but this section has now moved into the `SMTPConnectorImpl.Codeunit.al` in the `BuildSMTPMessage` function. 

Should be able to copy this part, paste it in there and update the variable names.

Reply:
Done.